File: Algorithm/Baekjoon/Java/기본_수학_1/분수찾기.java
package Algorithm.Baekjoon.Java.기본_수학_1;

import java.util.*;

public class 분수찾기 {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.close();
        int init = 1;
        int adder = 1;
        while (n > init) {
            adder += 1;
            init += adder;
        }
        int s = 0;
        int m = 0;
        if (adder % 2 == 0) {
            s = adder - (init - n);
            m = 1 + (init - n);
        } else {
            s = 1 + (init - n);
            m = adder - (init - n);
        }

        System.out.println(Integer.toString(s) + "/" + Integer.toString(m));
    }
}

File: Algorithm/Baekjoon/Python/기본 수학1/분수찾기.py
i = int(input())

init = 1
adder = 1
while i > init:
    adder += 1
    init = init + adder

if adder % 2 == 1:
    m = adder - (init - i)
    s = 1 + (init - i)
else:
    m = 1 + (init - i)
    s = adder - (init - i)

print("adder : " + str(adder))
print("init : " + str(init))
print(str(s) + "/" + str(m))
